### PR TITLE
Inquiry get request

### PR DESCRIPTION
**Pull request by jitendrajaria**
Inquiry requests added except getByStatus

### CODE REVIEW

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Resolve conflicts and resubmit PR

**Comment by jitendrajaria** [comment]
Issues Resolved in commit 47b61ff

File: src/main/java/creativei/helper/ResponseHelper.java [hunk 1]
```diff
@@ -56,23 +58,25 @@ private static AddressVo getCreateAddressResponseData(InquiryAddress inquiryAddr
         addressVo.setState(inquiryAddress.getState());
         return addressVo;
    }
-   private static List<EducationVo> getCreateEducationResponseData(List<InquiryEducation> inquiryEducation, List<EducationVo> educationVo) {
+   private static List<EducationVo> getCreateEducationResponseData(List<InquiryEducation> inquiryEducation, List<EducationVo> educationVos) {
+       EducationVo educationVo=new EducationVo();
        Iterator iterator = inquiryEducation.iterator();
        Integer i=0;
        while (iterator.hasNext()) {
```
Comment (aayushKhandpur) [inline src/main/java/creativei/helper/ResponseHelper.java:65]: Any reason why you used iterator instead of forEach loop?

**Review by aayushKhandpur — APPROVED** [review]
1. Merge Request mapping pincode branch first.
2. Reverse merge master in "Inquiry get request" branch.
3. Merge this branch into master.

**Pull request merged**